tests/robustness/coverage/README.md

```diff
@@ -39,10 +47,13 @@ tracing exporting to [Jaeger](https://www.jaegertracing.io/)
         jaegertracing/jaeger:2.6.0 --set=extensions.jaeger_storage.backends.some_storage.memory.max_traces=20000000
       ```
 
-   1. Run `etcd`:
+   1. Run `etcd` (in root of `etcd` repository):
 
       ```shell
-      etcd --watch-progress-notify-interval=5s \
+      make build
+      export KUBERNETES_REPO=$HOME/kubernetes # Change to the actual path to kubernetes repository.
```

Comment:
Don't repeat declaration of environment variable. Put it at the prerequisites for running script and suggest users to customize it if wanted.